_contentTemplates/common/general-info.md

```diff
@@ -48,21 +48,21 @@ You can reference the built-in Telerik assets from a cloud CDN instead of a loca
 
 
 #vsx-download
-You can get them from the:
+You can get them from:
 
-* [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=TelerikInc.TelerikBlazorVSExtensions) (for Windows)
-* [Telerik UI for Blazor automated installer]({%slug installation/msi%}) (for Windows and Mac)
-* Your [Telerik.com account](https://www.telerik.com/account/product-download?product=BLAZOR) - the `.mpack` file for VS for Mac
```

Comment:
Having "the" in the introductory sentence creates a conflict with "your" in the last bullet.